pkg/apply/merge.go

```diff
@@ -13,6 +13,7 @@ func MergeMetadataForUpdate(current, updated *uns.Unstructured) error {
 
 	mergeAnnotations(current, updated)
 	mergeLabels(current, updated)
+	updated.SetResourceVersion(current.GetResourceVersion())
```

Comment:
Why this is needed?

Reply:
During testing, I found some error messages during reconciliation. It complains fail to update the resource without the `resourceVersrion`.